workflow/sample_files/base-enclosure-other-housing-unit.xml

```diff
@@ -277,7 +284,7 @@
             <SystemIdentifier id='WaterHeater'/>
             <FuelType>electricity</FuelType>
             <WaterHeaterType>storage water heater</WaterHeaterType>
-            <Location>living space</Location>
+            <Location>other housing unit</Location>
```

Comment:
It seems like everything except duct location is set to 'other housing unit' in this file. We should also set duct location, right?